common/libs/inputValidator.js

```diff
@@ -0,0 +1,15 @@
+export function isInt(n) {
```

Comment:
@crptm I was using it for input validation on the swap form at first, but then I went with a different approach. Thought it might be useful to check in, but since you don't, I will just delete the whole file.